sw/device/lib/base/status_report_unittest_c.c

```diff
@@ -0,0 +1,37 @@
+// Copyright lowRISC contributors.
```

Comment:
do these have to be in a separate file? can they just be in the same  `tatus_report_unittest.cc`? then you don't need the header file either

Reply:
Yes, the `TRY` macro only works in C code because `INTO_STATUS` uses `_Generic` unfortunately.
I will see however if I can share this C because between the unit test and the "real" test.

Reply:
ah, I forgot about that, my apologies.